### PR TITLE
SSCSCI-12

### PR DESCRIPTION
**Pull request by hashimalisolirius**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCSCI-12

### Change description ###
Dependency suppression
Security supppression
Create Tests for PanelCompositionService

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -17,4 +17,14 @@
   <suppress until = "2023-09-01">
```
Comment (nitinprabhuhmcts) [inline config/owasp/suppressions.xml:17]: probably remove once which are not relevant anymore ?

File: src/test/java/uk/gov/hmcts/reform/sscs/service/PanelCompositionServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        CcdValue<OtherParty> otherParty = CcdValue.<OtherParty>builder()
+            .value(OtherParty.builder()
+                .id("1")
+                .hearingOptions(HearingOptions.builder().scheduleHearing("Yes").build())
+                .build())
+            .build();
+        sscsCaseData.setOtherParties(Collections.singletonList(otherParty));
+        sscsCaseData.setIsFqpmRequired(YesNo.YES);
+        panelCompositionService.processCaseState(callback,sscsCaseData, EventType.UPDATE_OTHER_PARTY_DATA);
+        assertNull(sscsCaseData.getDirectionDueDate());
```
Comment (benouaer) [inline src/test/java/uk/gov/hmcts/reform/sscs/service/PanelCompositionServiceTest.java:89]: where is directionDueDate set?

Comment (hashimalisolirius) [reply src/test/java/uk/gov/hmcts/reform/sscs/service/PanelCompositionServiceTest.java:89]: In the processCaseState method

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (hashimalisolirius) [reply config/owasp/suppressions.xml]: Have removed the old suppression

**Review by nitinprabhuhmcts — APPROVED** [review]

**Review by benouaer — APPROVED** [review]

**Pull request merged**